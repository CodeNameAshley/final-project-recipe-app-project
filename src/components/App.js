/* eslint-disable */
import React from "react";
import "../styles/App.css";
import SearchBar from "./SearchBar";
import SearchResults from "./SearchResults";
import data from "../data/data.json"

function App() {
  return (
    <div className='App'>
      <h1>Recipe App Project</h1>
      <SearchBar />
      <SearchResults data = {data}/>
    </div>
  );
}

export default App;
