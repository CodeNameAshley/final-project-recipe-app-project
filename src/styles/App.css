.App {
  text-align: center;
  background: linear-gradient(168.55deg, #d6eadf 0%, #edede8 50.00%, #e2ece9 100%);
}
