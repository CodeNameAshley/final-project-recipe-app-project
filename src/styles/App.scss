@import "../sass-styles/sass-variables/variables";
@import "../sass-styles/sass-variables/fonts";
@import "../sass-styles/sass-variables/colours";

.App {
  text-align: center;
  background: linear-gradient(168.55deg, #d6eadf 0%, #edede8 50.00%, #e2ece9 100%);
  @media screen and (max-width: $tablet-screen-size) {
    background: linear-gradient(168.55deg, #e8fffd 0%, #ffffec 50.00%, #f1fff8 100%);
  }

  ;
  @media screen and (max-width: $phone-screen-size) {
    background: linear-gradient(168.55deg, #f1fff8 0%, #ffffec 50.00%, #edfffa 100%);
  };
}
