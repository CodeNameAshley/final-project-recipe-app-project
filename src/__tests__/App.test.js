/* eslint-disable */
import { cleanup, render } from "@testing-library/react";
import App from "../components/App";


afterEach(cleanup)

it("Should take a snapshot", () => {
  const {asFragment} = render(<App />);
  
  expect(asFragment(<App />)).toMatchSnapshot()
});
