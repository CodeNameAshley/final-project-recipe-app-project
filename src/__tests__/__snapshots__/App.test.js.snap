// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navBar__main"
    >
      <a
        href="/"
      >
        <img
          alt="back button"
          class="back"
          src="back-button.png"
        />
      </a>
      <a
        href="/"
      >
        <img
          alt="foodle logo"
          src="foodle-logo.png"
        />
      </a>
      <a
        href="/login"
      >
        <p>
          login
        </p>
      </a>
    </div>
    <div
      class="search-bar__main"
    >
      <div
        class="search-chip-div"
      >
        <div
          class="chip__main"
        >
          <div
            class="chip-and-button"
          />
        </div>
        <form
          class="search-form"
        >
          <div
            class="search-bar__form-addButton"
          >
            <input
              class="search-input"
              id="inputField"
              placeholder="type your ingredients to find recipes"
              type="text"
              value=""
            />
            <button
              class="search-bar__add"
              type="button"
            >
              Add
            </button>
          </div>
          <button
            class="searchButton__disabled"
            type="submit"
          >
            Search
          </button>
        </form>
      </div>
    </div>
    <div
      class="search-results__main"
    >
      <header
        class="search-results__header"
      >
        ✨fancy trying something new?✨
      </header>
      <div
        class="search-results__card"
      >
        <p>
          Oh no there are no recipes in our cookbook!
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
